content-services/7.2/develop/repo-ext-points/content-transformers-renditions.md

```diff
@@ -218,6 +218,9 @@ changed by resetting the following Alfresco global property.
 local.transform.pipeline.config.dir=shared/classes/alfresco/extension/transform/pipelines
 ```
 
+Local transform-core-aio docker image might need a [newer version of image and approach](#adding-pipelines-and-failover-transforms-to-a-t-engine)
```

Comment:
Hey, sorry, I dont know whats going on here. Does this make sense? The link text doesnt make sense in the sentence does it? And I'm not sure about the use of Might.

It also links to a section that isnt called: newer version of image and approach.

Cheers.

Reply:
Thanks, I understand how it doesn't quite make sense. This is a line directly from the developer for this special case. I can change the link text, and hopefully everyone will accept the changes, and it will be easier to understand from a user's perspective.